src/AspNetCoreModuleV2/RequestHandler/environmentvariablehelpers.h

```diff
@@ -367,7 +367,7 @@ class ENVIRONMENT_VAR_HELPERS
     AddWebsocketEnabledToEnvironmentVariables
     (
         _Inout_ ENVIRONMENT_VAR_HASH*       pInEnvironmentVarTable,
-        _In_ BOOL                           fWebsocketsEnabled
+        _In_ WEBSOCKET_STATUS               fWebsocketsEnabled
```

Comment:
Why pass WEBSOCKET_STATUS   instead of bool? You don't try to initialize it in this method anyway.

Reply:
And if it's not initialized at this point it's an error.